src/Auth0.OidcClient.WinForms/WebViewBrowser.cs

```diff
@@ -51,6 +51,7 @@ public async Task<BrowserResult> InvokeAsync(BrowserOptions options, Cancellatio
             var tcs = new TaskCompletionSource<BrowserResult>();
 
             var window = _formFactory();
+            #pragma warning disable 618
```

Comment:
WebViewCompatible is obsolete, but we have no alternative on old platforms. For newer platforms, we already use the new version.

Ignoring this warning.